packages/modules/localstack/src/localstack-container.ts

```diff
@@ -0,0 +1,53 @@
+import { AbstractStartedContainer, GenericContainer, log, StartedTestContainer, Wait } from "testcontainers";
+
+export const LOCALSTACK_PORT = 4566;
+
+export class LocalstackContainer extends GenericContainer {
+  constructor(image = "localstack/localstack:2.2.0") {
+    super(image);
+  }
+
+  private resolveHostname(): void {
+    const envVar = "LOCALSTACK_HOST";
+    let hostnameExternalReason;
+    if (this.environment[envVar]) {
+      // do nothing
+      hostnameExternalReason = "explicitly as environment variable";
+    } else if (this.networkAliases && this.networkAliases.length > 0) {
+      this.environment[envVar] = this.networkAliases.at(this.networkAliases.length - 1) || ""; // use the last network alias set
```

Comment:
@ilopezluna this should call `withEnvironment` instead of making the assignment directly #834

Reply:
Thanks 🙏